src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -890,5 +890,82 @@ private static ErrorValue GetDiv0Error(IRContext irContext)
                 Message = "Division by zero"
             });
         }
+
+        private static FormulaValue Dec2Hex(IRContext irContext, NumberValue[] args)
+        {
+            var min_number = -Math.Pow(2, 39);
+            var max_number = Math.Pow(2, 39) - 1;
```

Comment:
>Math.Pow(2, 39) - 1 [](http://example.com/codeflow?start=29&length=19)

`(1 << 39) - 1` - bitwise operation is much more efficient than Math.Pow function